generate/g_appcode.go

```diff
@@ -255,7 +256,7 @@ func (tag *OrmTag) String() string {
 	if len(ormOptions) == 0 {
 		return ""
 	}
-	return fmt.Sprintf("`orm:\"%s\"`", strings.Join(ormOptions, ";"))
+	return fmt.Sprintf("`orm:\"%s\" description:\"%s\"`", strings.Join(ormOptions, ";"), tag.Comment)
```

Comment:
if tag.Comment is empty it will create the empty descriptio. Why you didn't follow the other logic, like Default.